app-backend/api/src/main/scala/stac/Routes.scala

```diff
@@ -42,8 +45,29 @@ trait StacRoutes
     (withPagination & stacExportQueryParameters) {
       (page: PageRequest, params: StacExportQueryParameters) =>
         complete {
+          val s3Client = S3()
           StacExportDao
             .list(page, params, user)
+            .map(p =>
+              PaginatedResponse[StacExport.WithSignedDownload](
+                p.count,
+                p.hasPrevious,
+                p.hasNext,
+                p.page,
+                p.pageSize,
+                p.results.map(export =>
+                  StacExport.signDownloadUrl(
```

Comment:
The logic here and below look similar. Would it be possible to abstract them and call the abstracted function here and below?
Also, I noticed that for old exports, they don't have a zipped catalog available in the signed url.